src/test/java/com/endava/internship/collections/StudentSetTest.java

```diff
@@ -0,0 +1,217 @@
+package com.endava.internship.collections;
+
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.*;
+import java.time.LocalDate;
+
+public class StudentSetTest {
+    private StudentSet set;
+
+    private final static Student st1 = new Student("John", LocalDate.of(2000, 5, 12), "");
+
+    private final static Student st2 = new Student("David", LocalDate.of(2001, 6, 20), "");
+
+    private final static Student st3 = new Student("Dmitrii", LocalDate.of(1996, 10, 16), "");
```

Comment:
rename please this constants to JOHN, DAVID and DMITRII

Reply:
Constants have been renamed